packages/nx/src/migrations/update-15-0-0/migrate-to-inputs.ts

```diff
@@ -90,11 +90,12 @@ export default async function (tree: Tree) {
               tree,
               join(project.root, 'package.json'),
               (json) => {
-                json.nx ??= {};
-                json.nx.namedInputs ??= {};
-                json.nx.namedInputs.projectSpecificFiles ??=
+                const packageJsonNxConfiguration =
+                  typeof json.nx === 'boolean' ? {} : json.nx ?? {};
```

Comment:
I don't think the whole nx being a boolean thing

Reply:
Let's chat about this one Monday morning